vitrina/projects/urls.py

```diff
@@ -38,6 +39,36 @@
         ProjectPermissionsView.as_view(),
         name="project-permissions",
     ),
+    path(
+        "projects/<int:pk>/client/",
```

Comment:
I recommend using full id names in path variables. Optional
```suggestion
        "projects/<int:project_id>/client/",
```